qparam_test.go

```diff
@@ -1,9 +1,11 @@
 // Copyright (c) 2017, A. Stoewer <adrian.stoewer@rz.ifi.lmu.de>
 // All rights reserved.
 
-package qparam_test
+package qparam
```

Comment:
If you run the tests in `qparam` instead of `qparam_test` you have to remove the import of `"github.com/stoewer/go-qparam"` below

Reply:
I just tried it without the `import .../go-qparam` and with Go 1.8.3, I get errors like the following:
`./qparam_test.go:85: undefined: qparam in qparam.NewReader`
when I run `go test *.go`

I have GOPATH set to ~/.go and checked out the repo into ~/.go/src/github.com/stoewer/go-qparam.

Reply:
Ant yet the tests on [travis](https://travis-ci.org/stoewer/go-qparam/jobs/288078815) wail with:

> import cycle not allowed in test
> package github.com/stoewer/go-qparam (test)
>	imports github.com/stoewer/go-qparam

You have to set package to `qparam_test` or remove the import `"github.com/stoewer/go-qparam"` - that's a fact. Therefore please change the package to `qparam_test`.